docs/requirements.md

```diff
@@ -0,0 +1,146 @@
+## Hydrophonitor CLI
+
+### Requirements
+
+Introduces a command line utility called `hp-cli` which interfaces with a Hydrophonitor, a Raspberry Pi based hydrophone passive acoustic monitoring system.
+
+### Supported commands
+
+#### 1. Import
+
+Introduces the following command:
+
+```sh
+hp-cli import DEVICE --output-path OUTPUT [options]
+```
+
+```
+Required arguments:
+
+DEVICE                      Path to USB mass storage or SD card where data will
+                            be imported from.
+
+--output-path OUTPUT        Path to where the directory for imported data
```

Comment:
I don't think this should be an option.

Reply:
I think output path should of course be an option. How do we know where the user wants to import the data to?

Reply:
I mean it doesn't need a flag, because it's a required argument.

```
hp-cli import /device/path /output/path --some-flag true

# as opposed to...

hp-cli import /device/path -o /output/path --some-flag true
```